src/osu_p2p.jl

```diff
@@ -37,17 +36,24 @@ function run_osu_p2p(benchmark::MPIBenchmark, func::Function, conf::Configuratio
         size = 1 << s
         iters = conf.iters(conf.T, s)
         # Measure time on current rank
-        time = func(conf.T, size, iters, comm)
+        time = func(conf.T, size, iters, comm, conf.window_size)
 
         if iszero(rank)
             # Number of bytes trasmitted
             bytes = size * sizeof(conf.T)
             latency = time / 2
 
+            if cal_bandwidth
+                tmp_total = bytes / 1e+6 * iters * conf.window_size
+                bandwidth = tmp_total / time
+                result = bandwidth
+            else
+                result = latency
+            end
```

Comment:
Can be slightly simplified to
```suggestion
            result = if cal_bandwidth
                tmp_total = bytes / 1e+6 * iters * conf.window_size
                tmp_total / time
            else
                latency
            end
```